app/models/work_usage.rb

```diff
@@ -0,0 +1,50 @@
+# class WorkUsage follows the model established by FileUsage
+# Called by the stats controller, it finds cached work pageview data,
+# and prepares it for visualization in /app/views/stats/work.html.erb
+
```

Comment:
You can remove this space, but otherwise I believe you've satisfied @jcoyne's request below. :+1: